packages/docs-theme/src/components/documentation.tsx

```diff
@@ -156,8 +168,13 @@ export class Documentation extends React.PureComponent<DocumentationProps, Docum
         // build up static map of all references to their page, for navigation / routing
         this.routeToPage = {};
         eachLayoutNode(this.props.docs.nav, (node, parents) => {
-            const { reference } = isPageNode(node) ? node : parents[0]!;
-            this.routeToPage[node.route] = reference;
+            if (isNavSection(node)) {
+                return;
```

Comment:
nit: Might be worth a comment that we never want to put the user on a "nav section" since these aren't click-able and don't have a real page. They're just presentational groupings of pages.